# clear your memory

In [1]:
clear all

# check if your Stata works fine

In [2]:
about


Stata/MP 14.2 for Unix (64-bit Intel)
Revision 29 Jan 2018
Copyright 1985-2015 StataCorp LLC

Total usable memory:   16.00 GB

Single-user 8-core Stata perpetual license:
       Serial number:  10699393
         Licensed to:  Ning Xu
                       Stark Industries


# Question 4

## $sleep = \beta_0 + \beta_1 \cdot totwrk + \beta_2 \cdot educ + \beta_3 \cdot age + u$

In [3]:
use "~/Data/Dropbox/Working_Directory/Tex_MD/Teaching/tutorial/6002/W1/2021S2/SLEEP75.DTA", clear

## (a) If adults trade off sleep for work, what is the sign of $\beta_1$?

### If adults trade off sleep for work, more work implies less sleep (other things equal), so $\beta_1 < 0$

## (b) What signs do you think $\beta_2$ and $\beta_3$ will have?

### The signs of and are not obvious. One could argue that more educated people like to get more out of life, and so, other things equal, they sleep less ($\beta_2 < 0$). The relationship between sleeping and age is more complicated than this model suggests.

## (c) regress sleep on totwrk, educ and age

In [4]:
reg sleep totwrk educ age


      Source |       SS           df       MS      Number of obs   =       706
-------------+----------------------------------   F(3, 702)       =     29.92
       Model |  15784778.6         3  5261592.87   Prob > F        =    0.0000
    Residual |   123455057       702  175861.905   R-squared       =    0.1134
-------------+----------------------------------   Adj R-squared   =    0.1096
       Total |   139239836       705  197503.313   Root MSE        =    419.36

------------------------------------------------------------------------------
       sleep |      Coef.   Std. Err.      t    P>|t|     [95% Conf. Interval]
-------------+----------------------------------------------------------------
      totwrk |  -.1483734   .0166935    -8.89   0.000    -.1811487   -.1155982
        educ |  -11.13381   5.884575    -1.89   0.059    -22.68729    .4196615
         age |   2.199885   1.445717     1.52   0.129    -.6385613    5.038331
       _cons |   3638.245   112.2751    32.40   0.

### Since totwrk is in minutes, we must convert five hours into minutes: $totwrk = 5 \times 60 = 300$. Then sleep is predicted to fall by $0.148 \times 300 = 44$ minutes. For a week, 45 minutes less sleep is not an overwhelming change.

## (d) Discuss the sign and magnitude of the estimated coeffiecient on educ. 

### More education implies less predicted time sleeping, but the effect is quite small. If we assume the difference between college and high school is four years, the college graduate sleeps about 45 minutes less per week, other things equal.

## (e) Would you say totwrk, educ, and age explain much of the variation in sleep? What other factors might affect the time spent sleeping? Are these likely to be correlated with totwrk?

### Not surprisingly, the three explanatory variables explain only about 11.3% of the variation in sleep. One important factor in the error term is general health. Another is marital status and whether the person has children. Health (however we measure that), marital status, and number and ages of children would generally be correlated with totwrk. (For example, less healthy people would tend to work less.)

In [5]:
!jupyter nbconvert --to html W1_Stata.ipynb


[NbConvertApp] Converting notebook W1_Stata.ipynb to html
[NbConvertApp] Writing 570123 bytes to W1_Stata.html
